cypress/integration/page.spec.js

```diff
@@ -1,7 +1,94 @@
-// WRITE TESTS HERE
+/// <reference types="Cypress" />
+
+const misha = {
+  name: 'Misha',
+  age: 37,
+  sex: 'm',
+  isMarried: true,
+  partnerName: 'Natasha',
+};
+
+const olya = {
+  name: 'Olya',
+  sex: 'f',
+  isMarried: true,
+  partnerName: 'Maksym',
+};
+
+const alex = {
+  name: 'Alex',
+  age: 25,
+  sex: 'm',
+  isMarried: false,
+};
 
 describe('Page', () => {
   it('should be visitable', () => {
     cy.visit('/');
   });
+
+  it('should have 3 persons', () => {
+    cy.getByDataCy('person')
+      .should('have.length', '3');
+  });
+
+  it(`first person name should be ${misha.name}`, () => {
+    cy.getByDataCy('person')
+      .selectElement('name', 0)
+      .should('contain', misha.name);
+  });
+
+  it(`first person's age should be ${misha.age}`, () => {
+    cy.getByDataCy('person')
+    .selectElement('age', 0)
+    .should('contain', misha.age);
```

Comment:
тут і в деяких місцях нижче недостатньо відступів
```suggestion
      .selectElement('age', 0)
      .should('contain', misha.age);
```